### PR TITLE
chore: remove lodash usages

### PR DESCRIPTION
**Pull request by anonrig**
Removes unnecessary Lodash usages. More information can be found from [You don't need Lodash](https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore) repository.

Removed usages:

- `lodash/isArray`
- `lodash/isString`

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -30,6 +30,8 @@ module.exports = {
   },
 
   rules: {
+    eqeqeq: 'off',
```
[the remark this answers is not in the document]
Comment (scttcper) [reply .eslintrc.js:33]: we also try to put rules in that sentry-app config to reuse in getsentry

[the remark this answers is not in the document]
Comment (anonrig) [reply .eslintrc.js:33]: > not sure our largely python org will remember to use triple vs double equals

I see this as an education opportunity, not a linter issue.

> we also try to put rules in that [sentry app](https://github.com/getsentry/eslint-config-sentry) config to reuse in getsentry

I opened a PR: https://github.com/getsentry/eslint-config-sentry/pull/193

**Review by billyvg — COMMENTED** [review]
Should we [disallow these in eslint](https://github.com/getsentry/eslint-config-sentry/blob/master/packages/eslint-config-sentry-app/index.js#L157-L161) so it doesn't regress?

**Comment by anonrig** [comment]
> Very much in favor of removing this and can help contribute. Do you accept contributions to the PR or should I open a separate branch and target this?

I accept contributions in any kind :-)

**Review by JonasBa — APPROVED** [review]

**Comment by ryan953** [comment]
refs: https://github.com/getsentry/frontend-tsc/issues/55

**Pull request merged**